lib/puppet/provider/reboot/windows.rb

```diff
@@ -110,7 +110,7 @@ def pending_file_rename_operations?
     with_key(path) do |reg|
       renames = reg.read('PendingFileRenameOperations') rescue nil
```

Comment:
Probably should have used a destructuring assignment here for readability (to avoid the array indexing), like

```ruby
rename_type, renames = reg.read('PendingFileRenameOperations') rescue nil
```